src/components/PaymentCardForm.vue

```diff
@@ -64,6 +64,7 @@ export default{
             let expMonth = document.getElementById("expMonth").value
             let expYear = document.getElementById("expYear").value
             let cvc = document.getElementById("cvc").value
+            //A ENLEVER
```

Comment:
You hsould use the `this.$refs` from VueJs instance instead of using document.getElementById in that case you'll be able to get the value.

However if you want a direct data binding you can proceed to something like this:

```javascript
<template>
<!-- Two way Binding example (usage of v-model) -->
 <your-component v-model="expMonth"></your-component>
<!-- refs system that can have the same result but with no double data binding -->
<input ref="expMonth" />
</template>

<script>
export default {
 data() {
   return {
     expMonth: null
   }
 }
}
</script>
```